src/concrete/ml/torch/hybrid_model.py

```diff
@@ -53,35 +81,107 @@ def convert_conv1d_to_linear(layer_or_module):
     return layer_or_module
 
 
+# pylint: disable-next=too-many-instance-attributes
 class RemoteModule(nn.Module):
     """A wrapper class for the modules to be done remotely with FHE."""
 
     def __init__(
         self,
-        module=None,
-        server_remote_address=None,
+        module: Optional[nn.Module] = None,
+        server_remote_address: Optional[str] = None,
+        module_name: Optional[str] = None,
+        model_name: Optional[str] = None,
+        verbose: int = 0,
```

Comment:
so we should have an Args section in this docstring

Reply:
Indeed, I'm surprised this isn't enforced by our docstring checks tbh

Reply:
I think init are exempted, and we never put args in __init__ (mostly because we didn't want to add init docstrings for built-in models, which would require copy pasting from sklearn)

Reply:
I added some documentation